### ISSUE TITLE
Lost playlists despite re-using old .quodlibet directory after distro switch

### ISSUE THREAD

**Issue by pschwede** [quodlibet/quodlibet#2296]
Now they are gone.

**Comment by pschwede** [comment]
Sorry. My fault -- I moved the files, hence their paths aren't contained in any playlist anymore.

**Comment by pschwede** [comment]
I adapted the content of the files in ~/.quodlibet/playlists to the current folder structure of my library. Still all playlists are displayed empty in Quod Libet. Why is that?

**Comment by pschwede** [comment]
I adapted the content of the files in ~/.quodlibet/playlists to the current folder structure of my library. Still all playlists are displayed empty in Quod Libet. Why is that?

**Comment by CreamyCookie** [comment]
@pschwede Just to be sure: Have you added this folder to QL (Preferences > Library > Scan Directories > Add), scanned the library (File > Scan Library) and closed QL before adding the playlists files?

**Comment by pschwede** [comment]
@CreamyCookie Not necessarily in that order, but yes.

I fixed it myself, after I took a look into some song's details. QL seems to ´readlink -f´ linked file paths. So I was able to fix my playlists using absolute and un-linked paths.

**Comment by CreamyCookie** [comment]
Good to hear! :)

**Comment by mistere357** [comment]
I'm a Linux command line user.
I set up a git repository in .quodlibet/playlists.  'git reset --hard HEAD' is your friend when Quod Libet re-writes the playlists to zero length. 
Sed allows me to re-write the paths in the playlists when the address of the library changes as it did recently when I switched from Fedora to KUbuntu Linux. Quod Libet does not care if it's linked to /home/eric/Music - it wants the actual mount point of the USB device where the music is. 
I agree that Quod Libet should warn the user if the playlists are going to be changed and offer the option to not continue with that process. I have 941 playlists and without git and sed I'd be hosed.
